src/odemis/driver/test/static_test.py

```diff
@@ -69,6 +69,22 @@ def test_mag_choices(self):
             comp.magnification.value = 2.0
         comp.terminate()
 
+    def test_configurations(self):
+        configurations = {"Mirror up": {"pole_pos": (458, 519),  "focus_dist": 0.5e-3},
+                          "Mirror down": {"pole_pos": (634, 652),  "focus_dist": -0.5e-3}}
```

Comment:
Maybe add an extra configuration, with a non existing attribute (eg, "booo"), to test the handling (=> it shouldn't fail, but just log a warning).
PS: no need to check that it logged a warning, because it's actually really hard, and not so important behaviour-wise.